run-setup

```diff
@@ -39,6 +39,8 @@ source ./venv/bin/activate
 # pyjq setup fails with errors similar to https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=917006
 # explicitly installing wheel fixes the error
 pip3 install wheel
+# upgrade setuptools for pyrchain to work
+pip3 install -U setuptools
```

Comment:
pip3 install -U setuptools>=40.1
Otherwise if you have a setuptools lower than 40, it would install nothing

Reply:
https://github.com/rchain/rchain-testnet-node/pull/17/commits/70ed739bac26bcb6ff383de9c062cb712b950ef8